cellengine/resources/experiment.py

```diff
@@ -274,13 +378,11 @@ def get_statistics(
         )
 
     @property
-    def scalesets(self) -> List[ScaleSet]:
+    def scalesets(self):
```

Comment:
Here and `get_scaleset` - why remove the return type?

Reply:
Same as above, with `Experiment`: incompatible return types; fixed